openspec/changes/governance-03-deterministic-agent-governance-loading/specs/agent-governance-loading/spec.md

```
@@ -0,0 +1,94 @@
+## ADDED Requirements
+
+### Requirement: Compact AGENTS bootstrap contract
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)